wca-guidelines.md

```diff
@@ -60,6 +60,9 @@ To be more informative, each Guideline is classified using one of the following
 - 2j2++) [EXAMPLE] If the WCA Delegate disqualifies a competitor during their third attempt in a round, only the third attempt and all following attempts in that event are disqualified, even if the circumstances that originated the disqualification occurred prior to this attempt.
 - 2k6+) [CLARIFICATION] WCA Delegates should only use their discretion to prevent competitors from being a severe detriment to the competition (e.g. wasting time and/or competition resources). Competitors should not be disqualified for a "poor" result when they are competing to the best of their abilities.
 - 2s+) [REMINDER] Special accommodations must be noted in the Delegate Report.
+- 2s++) [EXAMPLE] Competitors whose circumstances (e.g. religion) do not allow them to compete on an specific date or time may be allowed to compete in another period under the discretion of the WCA Delegate. The WCA Delegate must consider the well-being of the competition when attending such requests.
+- 2s+++) [EXAMPLE] Competitors whose circumstances prevent them from following the correct procedures may be helped to do them. (e. g. a competitor that can not use one of their hands to start the timer may receive assistance from the judge to start and stop the timer, or a competitor that can not listen may receive their inspection warnings in a different way).
+- 2s++++) [EXAMPLE] Competitors whose circumstances result in a severe impediment of following an event procedures may have some leniency applied when competing. (e. g. blind competitor using a second hand to feel the pieces in 3x3x3 One-Handed or using a second puzzle with special pieces to memorize in 3x3x3 Blindfolded). These cases must be informed and requested to the WCA Regulations Committee before being applied.
```

Comment:
```suggestion
- 2s++++) [EXAMPLE] Competitors whose circumstances result in it being impossible to follow regular competition procedures may have special accommodations. (e. g. blind competitor using a second hand to feel the pieces in 3x3x3 One-Handed or using a second puzzle with special pieces to memorize in 3x3x3 Blindfolded). These cases should be brought up to the WCA Regulations Committee before being applied.
```
This one might go better as a sub regulation of 2s or something. More discussion needed here

Reply:
I agree it's worth adding it as a sub-reg.

Reply:
2s++++ should definitely be a sub-regulation rather than an example in the guidelines